demo_grid/src/Grid/Query/ProductQueryBuilder.php

```diff
@@ -202,25 +202,27 @@ private function getQueryBuilder(array $filterValues): QueryBuilder
             ->addFilter(
                 'id_product',
                 'p.`id_product`',
-                SqlFilters::MIN_MAX
-            )
-            ->addFilter(
-                'price_tax_excluded',
-                'ps.`price`',
-                SqlFilters::MIN_MAX
-            )
-        ;
-
-        if ($isStockManagementEnabled) {
+                SqlFilters::WHERE_STRICT
+            );
+        if (version_compare(_PS_VERSION_, '8.0', '>=')) {
             $sqlFilters
                 ->addFilter(
-                    'quantity',
-                    'sa.`quantity`',
+                    'price_tax_excluded',
+                    'ps.`price`',
                     SqlFilters::MIN_MAX
-                )
-            ;
+                );
+            /*
+            if ($isStockManagementEnabled) {
+                $sqlFilters
+                    ->addFilter(
+                        'quantity',
+                        'sa.`quantity`',
+                        SqlFilters::MIN_MAX
+                    )
+                ;
+            }
+            */
```

Comment:
```suggestion
```

@leemyongpakvn maybe we can remove it if it's not needed?

Reply:
@kpodemski I guess the original author @PierreRambaud had a plan to do with `$isStockManagementEnabled` as he prepared at [ProductQueryBuilder.php#L172-L198](https://github.com/PrestaShop/example-modules/blob/master/demo_grid/src/Grid/Query/ProductQueryBuilder.php#L172-L198), so I'd prefer keep the StockManagement Filter in comment block for the future improvement.